### ISSUE TITLE
clearCache error: AbstractMatcher  icompatibility

### ISSUE THREAD

**Issue by raupie** [kaliop-uk/ezmigrationbundle#181]
```
> eZ\Bundle\EzPublishCoreBundle\Composer\ScriptHandler::clearCache

In ContentVersionMatcher.php line 12:
                                                                               
  Warning: Declaration of Kaliop\eZMigrationBundle\Core\Matcher\ContentVersio  
  nMatcher::matchAnd(array $conditionsArray, $content) should be compatible w  
  ith Kaliop\eZMigrationBundle\Core\Matcher\AbstractMatcher::matchAnd($condit  
  ionsArray)                                                                   
                                                                               

Script eZ\Bundle\EzPublishCoreBundle\Composer\ScriptHandler::clearCache handling the symfony-scripts event terminated with an exception

                                                                                   
  [RuntimeException]                                                               
  An error occurred when executing the "'cache:clear --no-warmup'" command:        
                                                                                   
                                                                                   
                                                                                   
                                                                                   
  In ContentVersionMatcher.php line 12:                                            
                                                                                   
    Warning: Declaration of Kaliop\eZMigrationBundle\Core\Matcher\ContentVersio    
    nMatcher::matchAnd(array $conditionsArray, $content) should be compatible w    
    ith Kaliop\eZMigrationBundle\Core\Matcher\AbstractMatcher::matchAnd($condit    
    ionsArray)                                                                     
                                                                
```

**Comment by gggeek** [comment]
Thx. Dupe of https://github.com/kaliop-uk/ezmigrationbundle/issues/179

**Comment by gggeek** [comment]
Fixed in 5.7.1

**Comment by SalvatorePollaci** [comment]
@gggeek Still getting this error in 5.7.1

```
www-data@af8301b78fd0:~/html/project/ezplatform$ COMPOSER_MEMORY_LIMIT=-1 composer update kaliop/ezmigrationbundle
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 0 installs, 1 update, 0 removals
  - Updating kaliop/ezmigrationbundle (5.4.1 => 5.7.1): Downloading (100%)
Package guzzle/guzzle is abandoned, you should avoid using it. Use guzzlehttp/guzzle instead.
Package symfony/assetic-bundle is abandoned, you should avoid using it. Use symfony/webpack-encore-pack instead.
Writing lock file
Generating autoload files
ocramius/package-versions:  Generating version class...
ocramius/package-versions: ...done generating version class

What about running composer thanks now?
This will spread some 💖  by sending a ★  to 194 GitHub repositories of your fellow package maintainers.

> Incenteev\ParameterHandler\ScriptHandler::buildParameters
Updating the "app/config/parameters.yml" file
> eZ\Bundle\EzPublishCoreBundle\Composer\ScriptHandler::clearCache

In ContentVersionMatcher.php line 12:

  Warning: Declaration of Kaliop\eZMigrationBundle\Core\Matcher\ContentVersio
  nMatcher::matchAnd(array $conditionsArray, $content = NULL) should be compa
  tible with Kaliop\eZMigrationBundle\Core\Matcher\AbstractMatcher::matchAnd(
  $conditionsArray)


Script eZ\Bundle\EzPublishCoreBundle\Composer\ScriptHandler::clearCache handling the symfony-scripts event terminated with an exception


  [RuntimeException]
  An error occurred when executing the "'cache:clear --no-warmup'" command:




  In ContentVersionMatcher.php line 12:

    Warning: Declaration of Kaliop\eZMigrationBundle\Core\Matcher\ContentVersio
    nMatcher::matchAnd(array $conditionsArray, $content = NULL) should be compa
    tible with Kaliop\eZMigrationBundle\Core\Matcher\AbstractMatcher::matchAnd(
    $conditionsArray)


update [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--lock] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [--with-dependencies] [--with-all-dependencies] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--prefer-stable] [--prefer-lowest] [-i|--interactive] [--root-reqs] [--] [<packages>]...
```
